fenix/app/src/test/java/org/mozilla/fenix/telemetry/TelemetryMiddlewareTest.kt

```diff
@@ -86,6 +86,8 @@ class TelemetryMiddlewareTest {
         val engine: Engine = mockk()
         every { engine.enableExtensionProcessSpawning() } just runs
         every { engine.disableExtensionProcessSpawning() } just runs
+        every { engine.isTranslationsEngineSupported(any(), any()) } just runs
```

Comment:
In response to this [test-apk-fenix-debug ](https://firefoxci.taskcluster-artifacts.net/XYIsG1d5Rc-CiBRcZtdaxA/0/public/reports/test/testFenixDebugUnitTest/index.html) failure. Seems resolved with this correction. (It reports a different test in the report, but the logs helped me narrow down to this file.)

Reply:
(I might be able to remove this with the new changes, I'll do one CI with and one CI without to see if it is still required.)

Reply:
Appears to be required,[ failure without change](https://firefoxci.taskcluster-artifacts.net/A1R_M40fRVSG_JQaHct0sA/0/public/reports/test/testFenixDebugUnitTest/index.html).